process/block/sovereign/incomingHeader/incomingEventsProc/eventProcScCall.go

```diff
@@ -0,0 +1,76 @@
+package incomingEventsProc
+
+import (
+	"math/big"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-core-go/data/smartContractResult"
+	"github.com/multiversx/mx-chain-core-go/data/sovereign"
+
+	"github.com/multiversx/mx-chain-go/process/block/sovereign/incomingHeader/dto"
+)
+
+type eventProcScCall struct {
+	*eventProcDepositTokens
+}
+
+// NewEventProcScCall creates a new event processor for sc call operations
+func NewEventProcScCall(args EventProcDepositOperationArgs) (*eventProcScCall, error) {
+	err := checkArgs(args)
+	if err != nil {
+		return nil, err
+	}
+
+	return &eventProcScCall{
+		&eventProcDepositTokens{
+			marshaller:    args.Marshaller,
+			hasher:        args.Hasher,
+			dataCodec:     args.DataCodec,
+			topicsChecker: args.TopicsChecker,
+		},
+	}, nil
+}
+
+func (ep *eventProcScCall) ProcessEvent(event data.EventHandler) (*dto.EventResult, error) {
```

Comment:
Missing comment. Please add comprehensive description, same as in other files :pray: